src/main/jhipster/src/main/webapp/app/entities/study/advance/abstract-advance.component.ts

```diff
@@ -113,12 +118,8 @@ export abstract class AbstractAdvanceComponent implements OnInit {
     }
 
     dismiss(advanceType: string) {
-        // Handle closing of modal when this page is loaded outside of Angular.
-        if ((<any>window.parent).closeModal) {
-            (<any>window.parent).closeModal(advanceType);
-        }
         if ((<any>window.parent)) {
-            (<any>window.parent).postMessage({ name: 'cancel', 'value': '' }, '*');
+            (<any>window.parent).postMessage({ name: SELECT_INSTANCES, advanceType }, '*');
         }
     }
```

Comment:
is the button labeled as "Back"? in my opinion a cancel/dismiss process should close everything, not start the process from scratch (i.e: start advance again by selecting instances)

Reply:
yes, the button is labeled as 'Back'. Currently, when the user clicks the 'Back' the instances selection modal is shown.